var/spack/repos/builtin/packages/py-azure-storage-blob/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAzureStorageBlob(PythonPackage):
+    """Microsoft Azure Blob Storage Client Library for Python"""
+
+    homepage = "https://github.com/Azure/azure-storage-python"
+    pypi = "azure-storage-blob/azure-storage-blob-12.9.0.zip"
+    maintainers = ['marcusboden']
+
+    version('12.9.0', sha256='cff66a115c73c90e496c8c8b3026898a3ce64100840276e9245434e28a864225')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-azure-core@1.10:', type=('build', 'run'))
+    depends_on('py-msrest@0.6.21:', type=('build', 'run'))
```

Comment:
Our package audits are reporting a few inconsistencies:

![Screenshot from 2022-01-03 10-52-38](https://user-images.githubusercontent.com/4199709/147917852-1d155a9e-920b-4fb5-bf20-de7a8f243a89.png)

Basically, this package has constraints that cannot be solved with the versions Spack knows about. Can you add proper versions to dependencies too?

Reply:
I have two PRs for these versions: #28207  and  #28208
I was planning to wait until they are merged. Or is there a better way?

Reply:
@marcusboden I see that these are already in #28208 and #28207 I made the dependency explicit in the description, feel free to submit a single PR next time.

Reply:
Will do! Thanks for the feedback!

Reply:
For future reference, we used to require that each package be updated in a separate PR, but we've relaxed this constraint for PRs that directly depend on each other. So you're welcome to continue to submit each change in separate PRs, but you no longer have to.